tagged-localbuild.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <manifest>
   <remote name="NXP" fetch="https://github.com/NXP/"/>
-  <remote name="aosp" review="https://android-review.googlesource.com/" fetch="https://android.googlesource.com/"/>
+  <remote name="aosp" fetch="https://android.googlesource.com/" review="https://android-review.googlesource.com/"/>
   <remote name="exfatprogs" fetch="https://github.com/exfatprogs/"/>
   <remote name="hybris" fetch="https://github.com/mer-hybris"/>
   <remote name="sony" fetch="https://github.com/sonyxperiadev/"/>
-
+  
   <default remote="aosp" revision="refs/tags/android-11.0.0_r46" sync-j="4"/>
-
+  
   <project name="QcRilAm" path="vendor/oss/qcrilam" remote="sony" revision="ef51ec65f5609e0fe9180ec3d2d054b72c0b8bee" upstream="master" dest-branch="master" groups="device"/>
```

Comment:
I kept those out on purpose since they add unnecessary white space.

Reply:
And add themselves to every future git diff when updating :P